components/GrantApplicationForm.tsx

```diff
@@ -39,6 +40,7 @@ export default function ApplicationForm() {
         const res = await fetchPostJSON('/api/sendgrid', data)
         if (res.message === 'success') {
           router.push('/submitted')
+          setSubmitted(true)
```

Comment:
You'll have to `setSubmitted` as soon as the button is pressed, not after the API call is successful.